src/main/java/dinkplugin/util/ItemUtils.java

```diff
@@ -121,16 +121,28 @@ public SerializedItemStack stackFromItem(ItemManager itemManager, int id, int qu
         return new SerializedItemStack(id, quantity, price, composition.getName());
     }
 
-    public String formatStack(SerializedItemStack item) {
-        return String.format("%d x %s (%s)", item.getQuantity(), item.getName(), QuantityFormatter.quantityToStackSize(item.getTotalPrice()));
+    public String formatStack(SerializedItemStack item, boolean includePrice) {
+        if (includePrice) {
+            return String.format("%d x %s (%s)", item.getQuantity(), item.getName(), QuantityFormatter.quantityToStackSize(item.getTotalPrice()));
+        }
+
+        return String.format("%d x %s", item.getQuantity(), item.getName());
     }
 
-    public Evaluable templateStack(SerializedItemStack item) {
+    public Evaluable templateStack(SerializedItemStack item, boolean includePrice) {
+        if (includePrice) {
+            return Replacements.ofMultiple("",
+                Replacements.ofText(String.valueOf(item.getQuantity())),
+                Replacements.ofText(" x "),
+                Replacements.ofWiki(item.getName()),
+                Replacements.ofText(" (" + QuantityFormatter.quantityToStackSize(item.getTotalPrice()) + ")")
+            );
+        }
+
         return Replacements.ofMultiple("",
             Replacements.ofText(String.valueOf(item.getQuantity())),
             Replacements.ofText(" x "),
-            Replacements.ofWiki(item.getName()),
-            Replacements.ofText(" (" + QuantityFormatter.quantityToStackSize(item.getTotalPrice()) + ")")
+            Replacements.ofWiki(item.getName())
         );
     }
```

Comment:
another approach (up to you):

```java
    public Evaluable templateStack(SerializedItemStack item, boolean includePrice) {
        List<Evaluable> parts = new ArrayList<>(
            Arrays.asList(
                Replacements.ofText(String.valueOf(item.getQuantity())),
                Replacements.ofText(" x "),
                Replacements.ofWiki(item.getName())
            )
        );
        if (includePrice) {
            parts.add(Replacements.ofText(" (" + QuantityFormatter.quantityToStackSize(item.getTotalPrice()) + ")"));
        }
        return JoiningReplacement.builder().components(parts).build();
    }
```

Reply:
I think I prefer the basic code-duplication we have now